### PR TITLE
Update GDPR section on docs

### PR DESCRIPTION
**Pull request by birthe**

### CODE REVIEW

**Comment by birthe** [comment]
@oliverguenther I have moved some parts of the current statement on security and privacy (https://www.openproject.org/gdpr-compliance/) to the docs. The rest of the page will be updated soon and then link to it. Could you please quickly have a look? Thank you

File: docs/enterprise-guide/enterprise-cloud-guide/GDPR/README.md [hunk 1]
```diff
@@ -13,6 +13,54 @@ The General Data Protection Regulation (GDPR) is a European regulation to harmon
 
 As a firm believer in open-source, OpenProject is invested heavily in the freedom of users. This encompasses the software freedoms granted by the [GPLv3](https://www.gnu.org/licenses/quick-guide-gplv3.en.html) and employed by OpenProject and naturally extends to the rights and  freedoms granted by the General Data Protection Regulation (GDPR). In  the same transparent fashion that we develop our software, we are  committed to transparency regarding data privacy protection of our users.
 
+## Information Security and Compliance
+
+### Hosting infrastructure
+
+OpenProject cloud environment are hosted on a logically isolated virtual cloud at Amazon Web Services with all services being located in Europe. AWS is a [GDPR compliant](https://aws.amazon.com/compliance/gdpr-center/) cloud infrastructure provider [with extensive security and compliance programs](https://aws.amazon.com/security/) as well as unparalleled access control mechanisms to ensure data privacy. Employed facilities are compliant with the ISO 27001 and 27018 standards.
```
Comment (oliverguenther) [inline docs/enterprise-guide/enterprise-cloud-guide/GDPR/README.md]:
```suggestion
OpenProject cloud environment is hosted on a logically isolated virtual cloud at Amazon Web Services with all services being located in Europe. AWS is a [GDPR compliant](https://aws.amazon.com/compliance/gdpr-center/) cloud infrastructure provider [with extensive security and compliance programs](https://aws.amazon.com/security/) as well as unparalleled access control mechanisms to ensure data privacy. Employed facilities are compliant with the ISO 27001 and 27018 standards.
```

File: docs/enterprise-guide/enterprise-cloud-guide/GDPR/README.md [hunk 2]
```diff
@@ -13,6 +13,54 @@ The General Data Protection Regulation (GDPR) is a European regulation to harmon
...
+
+**Hosting in Germany (on request)**
+
+We offer secure hosting of your OpenProject cloud also in a German data center on request. Please [contact us](https://www.openproject.org/contact-us/).
+
+### Data backups
+
+OpenProject cloud environment is continuously backing up user data with data at rest being fully encrypted with AES-256. Each individual instance is logically separated and data is persisted in a unique database schema, reducing the risk of intersection or data leaks between instances.
+
+### Access **to** data and infrastructure
```
Comment (oliverguenther) [inline docs/enterprise-guide/enterprise-cloud-guide/GDPR/README.md]:
```suggestion
### Access to data and infrastructure
```

Removing the bold tag within headline

File: docs/enterprise-guide/enterprise-cloud-guide/GDPR/README.md [hunk 3]
```diff
@@ -13,6 +13,54 @@ The General Data Protection Regulation (GDPR) is a European regulation to harmon
...
+
+OpenProject provides means to fully erase both all identifiable information of a user from the application. If the user is still referenced from data within the instance, these references are replaced with an anonymous user to ensure the data integrity of the application.
+
+- Data controllers can perform the deletion [through the administration](../../../system-admin-guide/users-permissions/users/).
+
+- Depending on the configuration of your OpenProject instance, individual data subjects may perform the deletion of their own account through the [Delete Account](../../../getting-started/my-account/) page. If this is disabled, the request may be stated to the data controller.
+
+### Data Portability
+
+OpenProject provides means to data controllers in order to receive *all* personal data connected to the OpenProject instance. This encompasses all user and system data (in the form of an SQL dump) as well as a collection of all uploaded files.
```
Comment (oliverguenther) [inline docs/enterprise-guide/enterprise-cloud-guide/GDPR/README.md:62]: This is now possible by controllers on their own using the backup feature of OpenProject: https://www.openproject.org/docs/enterprise-guide/enterprise-cloud-guide/backups/

Could you add that to the statement?

**Review by oliverguenther — CHANGES_REQUESTED** [review]
Hi @birthe I have some minor remarks, good to merge otherwise :+1:

**Comment by birthe** [comment]
Thanks a lot @oliverguenther

**Pull request merged**